x-pack/plugins/session_view/public/components/SessionView/index.tsx

```diff
@@ -123,7 +129,31 @@ export const SessionView = ({ sessionEntityId, height }: SessionViewDeps) => {
             onProcessSelected={onProcessSelected}
           />
         </div>
-      )}
+      );
+    }
+    return (
```

Comment:
should we explicitly check for error? instead of falling into this.

Reply:
right, we should